### DISCUSSION TITLE
Testing new discussion

### DISCUSSION THREAD

**Discussion by AAkindele** [retaildevcrews/ngsa#554] [PnP Deploy]
Testing new discussion

**Comment by AAkindele** [comment]
test comment

**Reply by AAkindele** [reply]
test reply to first comment

**Reply by AAkindele** [reply]
another one

**Comment by AAkindele** [comment]
test second comment

**Comment by chenmliu** [comment]
@Natasha-Kohli @AAkindele @alkofahi  @gled4er  testing tagging. Did you get a notification?

**Reply by Natasha-Kohli** [reply]
I didn't see any notification on Github, but I did get an email to my personal account linked to my GitHub account.
Also interesting that this seems to be the same discussion post on our [FC:PnPDeploy page](https://github.com/orgs/retaildevcrews/teams/fc-pnp-deploy), but I'm unable to see comments there. So instead [this view](https://github.com/retaildevcrews/ngsa/discussions) is better to check.

**Comment by AAkindele** [comment]
@chenmliu 👍 Yep. I got a notificationn
